x-pack/solutions/search/plugins/search_synonyms/server/routes.ts

```diff
@@ -270,4 +273,158 @@ export function defineRoutes({ logger, router }: { logger: Logger; router: IRout
       });
     })
   );
+
+  router.put(
```

Comment:
Question: Were there API tests added for these new routes? If so, can you please link them?

Reply:
a few of the new routes are missing tests due to the time constraints. I will add them later on.

Reply:
We are using jira in our team, I will and let you know